app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -141,6 +143,15 @@ private fun ConversationScreen(
     }
 }
 
+@Composable
+fun getErrorMessage(errorCode: ConversationErrors) =
+    when (errorCode) {
+        ErrorMaxAssetSize -> stringResource(R.string.error_conversation_max_asset_size_limit)
+        ErrorMaxImageSize -> stringResource(R.string.error_conversation_max_image_size_limit)
+        ErrorSendingSize -> stringResource(R.string.error_conversation_sending_image)
```

Comment:
I'm wondering if this `ErrorSendingSize` should be named to `ErrorSendingAsset` since this is the actual response from the server 🤔

Reply:
The asset limit exists even in the BE, the only problem I see with not going with solution 2 is DB migration later will not be easy.